### PR TITLE
Team added

### PR DESCRIPTION
**Pull request by Mr-Magnificent**
The "lines change" is bloated because of linter that automatically formatted the documents. :neutral_face:

### CODE REVIEW

File: web/templates/footer.html [hunk 1]
```diff
@@ -24,6 +24,5 @@ <h4>Email</h4>
 </div>
 <div class="container-fluid foot-credits">
   <div class="col">
-      <p>Copyright &copy; 2018, IEEE MSIT <br> Designed and Developed by <a href="https://www.linkedin.com/in/sahilkhurana19/" target="_blank">Sahil Khurana</a>, <a href="https://ankushgarg1998.github.io/">Ankush Garg</a> & <a href="https://www.linkedin.com/in/pranayberry/" target="_blank">Pranay Berry</a></p>
-  </div>
+      <p>Copyright &copy; 2018, IEEE MSIT <br> Designed and Developed and Maintained by <a href="/team">MSIT website Team</a>
```
Comment (sahilkhurana19) [inline web/templates/footer.html]: IEEE MSIT Website Team instead of MSIT website team

File: web/templates/team.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %}
+{% load static %}
+{% block navbar %}
+{% include "navbar.html"%}
+{% endblock %}
+{% block title %}
+TEAM MSIT
```
Comment (sahilkhurana19) [inline web/templates/team.html]: Website Team*

File: web/views.py [hunk 3]
```diff
@@ -61,3 +61,9 @@ def health(request):
 def tpe(request):
     context = {}
     return render(request, 'tpe.html', context)
+    
+def team(request):
+	team = Team.objects.all()
+	context = {'team': team}
+	print(context)
```
Comment (sahilkhurana19) [inline web/views.py]: Please remove this

Comment (Mr-Magnificent) [reply web/views.py]: This is used to get data from database team table. Removed the print() statement tho.

File: web/views.py [hunk 4]
```diff
@@ -64,6 +64,5 @@ def tpe(request):
     
 def team(request):
 	team = Team.objects.all()
-	context = {'team': team}
-	print(context)
+	context = {'team': teamprint(context)
```
Comment (sahilkhurana19) [inline web/views.py]: Can you please verify this statement again?

**Pull request merged**